### PR TITLE
Markdown support for grants

### LINKED ISSUE #6307: Support markdown formatting in Gitcoin Grants posts in the Activity tab
It would be great if markdown formatting were supported in Gitcoin Grants posts in the Activity tab. Right now, it ends up looking odd if you try to use multiple newlines and hyperlinks in a post.

Screenshot 2020-03-26 12 03 04

### PR DESCRIPTION
**Pull request by iamonuwa**
##### Description
Adds markdown support to grants activity tab

##### Refers/Fixes
closes #6307

##### Testing

Write text in markdown format within a grants detail page.

https://www.loom.com/share/bdb2a1e07d1d48f4ae0c04f742233cd9

### CODE REVIEW

File: app/assets/v2/js/activity.js [hunk 1]
```diff
@@ -918,23 +918,23 @@ $(document).ready(function() {
...
+    //     new_text = new_text.replace(/&lt/g, '_');
+    //     new_text = new_text.replace(/\</g, '_');
+    //     new_text = new_text.replace(/\>/g, '_');
+    //     new_text = new_text.replace(/\n/g, '<BR>');
+    //     new_text = urlify(new_text);
+    //     new_text = linkify(new_text);
+    //     $(this).html(new_text);
+    //     $(this).addClass('clean');
+    //   }
+    // });
```
Comment (thelostone-mc) [inline app/assets/v2/js/activity.js:937]: ^ remove code which is no longer needed

**Comment by iamonuwa** [comment]
Thanks @thelostone-mc . Issue said it's for grants activity. I can make it work for the other places as well.

**Comment by iamonuwa** [comment]
@danlipert can I fix this #4013 as part of this?

**Comment by thelostone-mc** [comment]
@iamonuwa if that issue persists -> yes please ! a fix is always welcome

**Comment by iamonuwa** [comment]
> @iamonuwa if that issue persists -> yes please ! a fix is always welcome

The requirements looks different. I'll work on it as a separate PR @thelostone-mc.

Thanks

**Comment by thelostone-mc** [comment]
@iamonuwa let us know when this is good for review

**Comment by iamonuwa** [comment]
> @iamonuwa let us know when this is good for review

This ready for review.

File: requirements/base.txt [hunk 2]
```diff
@@ -93,5 +93,5 @@ redis==3.3.11
 pandas
 wiki
 django-bulk-update
-elasticsearch
```
Comment (thelostone-mc) [inline requirements/base.txt:96]: @iamonuwa you need to be a lil careful while adding in code to ensure you don't accidentally delete something else

**Comment by thelostone-mc** [comment]
@iamonuwa code looks sane but could you add an activity in grants and also show a screenshot on Townsquare how it renders?

If that's there -> I'm good approving this in

**Comment by iamonuwa** [comment]
@thelostone-mc I made new changes for 
kudos, townsquare and profile.

https://www.loom.com/share/0960f2a7c9544e88bcc3623c791300ee

**Comment by thelostone-mc** [comment]
@iamonuwa video looks good !
Though you might have cherry-pick your commits from the looks of it ! It says 273 commits + lines changes

image

**Comment by iamonuwa** [comment]
🤦‍♂️ had no idea the commits pulled others along. Let me fix this

**Pull request closed without merging**